src/ui/components.tsx

```diff
@@ -79,6 +81,15 @@ export const CheckMkGenericAsyncSelect = function <Value extends string | (strin
     async (inputValue: string): Promise<Array<SelectableValue<Value>>> => {
       try {
         const data = await autocompleter(inputValue);
+        if (showVariables !== false) {
```

Comment:
That means, this branch should run if `showVariables` is undefined, i.e. left out when the component is instantiated? I'm sure about the logic here.

Reply:
added a comment at the prop definition for clarification.
most of the cases the variables should be shown, but there are some exceptions where `showVariables` is explicitly set to false.